Replace dir -> folder (#26)
seqeralabs/nf-training-intro: docs/part1-cli.md

   cd cli
   ```

> 3. You can check you are in the correct folder by using the [`pwd`](https://en.wikipedia.org/wiki/Pwd) (print working folder) command:

Reviewer: ```suggestion
3. You can check you are in the correct folder by using the [`pwd`](https://en.wikipedia.org/wiki/Pwd) (print working directory (folder)) command:
```

For simplicity, let's use the same labels, classify each of the 8 images individually and then copy them into a folder based on the classification:

> 1. Precreate individual folders for each label with the [`mkdir`](https://en.wikipedia.org/wiki/Mkdir) (folder folder) command:

Reviewer: ```suggestion
1. Precreate individual folders for each label with the [`mkdir`](https://en.wikipedia.org/wiki/Mkdir) (make directory (folder)) command:
```

> 1. Precreate individual folders for each label with the [`mkdir`](https://en.wikipedia.org/wiki/Mkdir) (make directory (folder)) command:

Reviewer: ```suggestion
1. Precreate individual folders for each label with the [`mkdir`](https://man7.org/linux/man-pages/man1/mkdir.1.html) (make directory (folder)) command:
```

Now you should have 3 directories labelled by critter:

1. A `dog/` folder with all of the images that were labelled as `'dog'`
2. A `cat/` folder with all the images labelled as `'cat'`
3. A `cute_dog/` folder with all the images labelled as `'cute_dog'`

You can also list the images that ended up in each directory by using the command below:

```bash
ls dog
```

> ## Step 3: folder a collage for each classifier

Reviewer: ```suggestion
## Step 3: Make a collage for each classifier
```

Now let's try to create a collage of each of the directories containing label-specific images. We will be using a tool called [`montage`](https://imagemagick.org/script/montage.php) to create the collage. This tool will take as input a list of images and output a collage in PNG format.

1. Create a `collages` folder:

   ```bash
   mkdir collages
   ```

> 2. Create a temporary image to folder a collage with the correct layout for all of the images in the `'dog'` folder:

Reviewer: ```suggestion
2. Create a temporary image to make a collage with the correct layout for all of the images in the `'dog'` folder:
```

> You can view the results of this command by clicking the `dog_temp.png` filename in the file explorer on the left-hand side of your window:
> ![open collage](assets/open_dog_temp.png)

> To folder the collage a little bit more interesting, we can add a few command-line arguments:

Reviewer: ```suggestion
To make the collage a little bit more interesting, we can add a few command-line arguments:
```
